docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>it's still kindof strange this is commented out</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>